Cargo.toml

```diff
@@ -36,3 +37,15 @@ smol_str = "0.2.2"
 annotate-snippets = "0.11.5"
 num-bigint = "0.4.6"
 if_chain = "1.0.2"
+
+[patch.crates-io]
```

Comment:
please include documentation blocks from Scarb/CairoLS so that it will be clear what we're doing here

Reply:
don't forget about the comment for `[dependencies]` 😉